bootstrap4/.js/src/test/scala/io/udash/bootstrap/utils/UdashIconsTest.scala

```diff
@@ -1,15 +1,21 @@
 package io.udash.bootstrap.utils
 
-import io.udash._
-import io.udash.css.CssView
-import io.udash.testing.UdashCoreFrontendTest
-
-
-class UdashIconsTest extends UdashCoreFrontendTest with CssView {
-
-  import scalatags.JsDom.all._
-
+import com.avsystem.commons.concurrent.RunNowEC
+import io.udash.bootstrap.UdashBootstrap
+import io.udash.bootstrap.utils.UdashIcons.FontAwesome._
+import io.udash.css.{CssStyle, CssView}
+import io.udash.properties.single.Property
+import io.udash.testing.AsyncUdashCoreFrontendTest
+import org.scalajs.dom._
+import org.scalatest.Succeeded
+import scalatags.JsDom.all._
+
+import scala.concurrent.Future
+
+class UdashIconsTest extends AsyncUdashCoreFrontendTest with CssView {
   "Bootstrap icons" should {
+    document.body.appendChild(UdashBootstrap.loadFontAwesome())
```

Comment:
I'd rather this was in `beforeAll`, no guarantees about when the constructor is called here at all